doc/rbd/rbd-kubernetes.rst

```diff
@@ -207,9 +207,9 @@ provision a `PersistentVolume`, which would be backed by a Ceph block image.
 An optional `volumeMode` can be included to select between a mounted file system
 (default) or raw block device-based volume.
 
-Using `ceph-csi`, specifying `Filesystem` for `volumeMode` can support both
-`ReadWriteOnce` and `ReadOnlyMany` `accessMode` claims, and specifying `Block`
-for `volumeMode` can support `ReadWriteOnce`, `ReadWriteMany`, and
+Using `ceph-csi`, specifying `Filesystem` for `volumeMode` can support all
+`ReadWriteOnce`, `ReadOnlyMany`, and `ReadWriteMany` `accessMode` claims, and
```

Comment:
The RBD `ceph-csi` driver does not support `ReadWriteMany` under `Filesystem` volume mode -- under `Block` mode `ReadWriteMany` is supported.

Reply:
Yes, as it should be.

Reply:
... am I missing something? Your change is incorrect -- you are saying that somehow ext4 or XFS file systems are supported on RBD block devices under ReadWriteMany?

Reply:
Sorry, You're right. I just take a look at ceph-csi support matrix for rbd and cephfs. Previously, I had confused cephfs ceph-csi with rbd ceph-csi on filesystem volume mode. I'm closing this PR.